doc/release/platforms/udp.rst

```diff
@@ -0,0 +1,161 @@
+.. _using-udp:
+
+==============================
+Using the Portable UDP Conduit
```

Comment:
Yeah, but you'll need to explicitly label platforms in the platforms/index.rst rather than relying on the `**` to automatically include them all. I don't think this is a huge maintainability issue, since we don't add/remove platforms all that often, and developers and will be notified of the error when a non-existent file is included, or existent file is not included.